CMGTools/TTHAnalysis/python/analyzers/ttHhistoCounterAnalyzer.py

```diff
@@ -32,23 +35,74 @@ def beginLoop(self, setup):
             setup.services["outputfile"].file.cd()
             self.inputCounter = ROOT.TH1D("Count","Count",1,0,2)
             if self.cfg_comp.isMC:
+                self.maxSMSmass = getattr(self.cfg_ana, 'SMS_max_mass', 3000)
+                self.inputCounterSMS = ROOT.TH3D("CountSMS","CountSMS",int(self.maxSMSmass+1),-0.5,self.maxSMSmass+0.5,int(self.maxSMSmass+1),-0.5,self.maxSMSmass+0.5,1,0,2)
```

Comment:
shouldn't the creation of theis TH3 go into `initSMS`? (and the same for the following ones)